shared/common-adapters/button.stories.js

```diff
@@ -64,6 +69,38 @@ const load = () => {
         <Button {...commonProps} type="PrimaryGreenActive" label="Primary Green Active" />
         <Button {...commonProps} type="PrimaryGreenActive" label="Primary Green Active" disabled={true} />
       </Pair>
+      <Pair background={globalColors.red}>
+        <Button {...commonProps} type="PrimaryColoredBackground" backgroundMode="Red" label="Primary Colored Background Red" />
+        <Button {...commonProps} type="PrimaryColoredBackground" backgroundMode="Red" label="Primary Colored Background Red" disabled={true} />
```

Comment:
Per the style guide, the disabled buttons should have white text: https://zpl.io/VkKl3Jg

Otherwise, this looks good to me.

Reply:
@adamjspooner that's for secondary buttons, which are correctly white. The style guide has no specific mention of how these buttons should look disabled, except for the general "30% opacity" guidance.

Reply:
🤦‍♂️ Next time I'll read the words on the buttons.